lab-judy/app/view/home/home.html

```diff
@@ -0,0 +1,30 @@
+<main class="home">
+  <nav class="gallery-list-container">
+    <h2> <span ng-if="homeCtrl.galleries[0]"> {{ homeCtrl.galleries[0].username }}'s</span> galleries</h2>
+
+    <input
```

Comment:
This should be abstracting into a searchbar component, with its own controller and template, to which you pass appropriate bindings from the homeCtrl. You'll need to place 2 <searchbar>s in here, one into which you pass the name and one into which you pass the description.